main.go

```diff
@@ -15,7 +15,11 @@ import (
 )
 
 func openLink(URL string) error {
-	return browser.OpenURL(URL)
+	if err := browser.OpenURL(URL); err != nil {
```

Comment:
note here, that having a single-return function with an error will now fail because you need to specify the error context. But that's good, it means more context and clarity.